bulk_processing/validators.py

```diff
@@ -36,6 +36,14 @@ def validate(value, **_kwargs):
     return validate
 
 
+def max_length(max_len: int):
```

Comment:
this might conflict with the new address stuff - there's a max_length function in that too. Merge with care